File: Web/Bar_Web_API/Bar_Web_API/Model/Shift.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Bar_Web_API.Model
{
    public class Shift
    {
        public int Id { get; set; }
        public DateTime DateTime { get; set; }
        public float ShiftDurationHours { get; set; }
        public ShiftStatus ShiftStatus{ get; set; }
        public User Worker { get; set; }
    }
}

File: app/src/main/java/hr/fvlahov/barsapp/model/Shift.kt
package hr.fvlahov.barsapp.model

import hr.fvlahov.barsapp.utils.DateSerializer
import kotlinx.serialization.Contextual
import kotlinx.serialization.Serializable
import java.util.*

@Serializable
data class Shift(
    var id: Int,
    @Serializable(with = DateSerializer::class)
    var dateTime: Date,
    var shiftDurationHours: Float,
    var shiftStatus: ShiftStatus,
    var worker: User?
) {
}